scripts/common.sh

```diff
@@ -98,6 +99,34 @@ SUBMODULES_FOR_JX_PARALLEL=( # [jx-research] Virtual Machine
     ## Simulation:
     # "velodyne_simulator"
 )
+
+SUBMODULES_FOR_Koda_Laptop=(
```

Comment:
Please copy SUMMIT config `SUBMODULES_FOR_SUMMIT`

Reply:
In case @CazesC wants to emulate the Lidar behavior in Gazebo to simulate using Summit pose-estimates, he needs the `velodyne_simulator`. The only reason he needs to follow Summits' ROS-&-workspace set-up process is because he wants to set up our Git-management of submodules in Ubuntu 18, and not because he is trying to mimic Summits' ROS-&-workspace requirements.